### ISSUE TITLE
Documentation section on the web page

### ISSUE THREAD

**Issue by bergos** [rdfjs/rdfjs.github.io#22]
There is a [documentation section](https://github.com/rdfjs/rdfjs.github.io/blob/84f7354747ff00d62667cf80d62bc3c6347d43b6/index.html#L47-L48) on the page which is currently hidden because it doesn't contain any items. We shifted the discussion about which content it should contain in the PR discussion. I would like to open the discussion now on what scope and requirements we want to define for content in that section.

One question that I have in mind: Do we want library/framework specific documentation in there? For example, [Comunica](https://comunica.dev/docs/) has a lot of documentation, [RDF-Ext](https://rdf-ext.org/) documentation is currently in progress. But they are also included in the library list. There are also pages like https://rdfjs.vercel.app/ which cover multiple libraries/frameworks.

**Comment by rubensworks** [comment]
I don't have a strong opinion on this, but pages such as https://rdfjs.vercel.app/ covering multiple libs/frameworks can definitely fit in there IMO.

I'm not sure about lib-specific docs, since we could then have a link for every lib/framework. Would the READMEs of libs/framework fall under the docs definition in this case? Alternatively, we could add an optional docs entry for those libs/frameworks that have dedicated docs (aside from the README).

**Comment by bergos** [comment]
Yes, we could distinguish between simple READMEs and dedicated docs. The dedicated docs have some additional value and, therefore, should be listed there as well. But I also want to avoid that we end up with a copy of the library list.

I would keep all docs in the same section. Tags and a filter similar to the one in the libraries could be used to filter different kinds of docs.

**Comment by megoth** [comment]
I'll let you know once https://rdfjs.vercel.app/ has been released. I would be honored to have it listed on your site ^_^

**Comment by megoth** [comment]
Hey, https://rdfjs.vercel.app/ is now released for the public, and if you want to add to your website I would be honored ^_^

**Comment by bergos** [comment]
@megoth Please create a PR. Just copy one of the library entries (`@id` pointing to your page, `name`, and `description`. You can skip `implements`) to the [documentations](https://github.com/rdfjs/rdfjs.github.io/blob/f3a89d8fc08e7be12f677d777ada3b9e3eab1fd0/index.html#L51) section and adapt it.
